build/scripts/migrate.sh

```diff
@@ -16,6 +16,13 @@ if [ -z "${CENT_ETHEREUM_CONTRACTS_DIR}" ]; then
     CENT_ETHEREUM_CONTRACTS_DIR=${PARENT_DIR}/build/centrifuge-ethereum-contracts
 fi
 
+echo "sourcing in nvm"
+. $NVM_DIR/nvm.sh
+
+# Migrate logic is based on an old version of zeppelin_os and transient dependencies
+current_node=$(node -v)
+nvm use v10
```

Comment:
We do here `nvm use v10` here and below `nvm use $current_node`?

Reply:
Since we are forced to use node 10 in this steps of the eth deployment, this code just reverts to what it was set before it needed to use v10.

Reply:
Ah ok I see 👍